### DISCUSSION TITLE
Magnet/screwholes

### DISCUSSION THREAD

**Discussion by mps39** [kennetek/gridfinity-rebuilt-openscad#21] [Q&A]
Is there any way to reverse the countersink/counterbore holes for the baseplates? I'd rather put screws in from the top. Apologies if I'm missing something, but I don't understand why the countersink/counterbores are on the bottom. If I am missing something, please help me learn.

**Comment by kennetek** [comment]
They are for bolting top-heavy Gridfinity models to the baseplate to prevent them from falling over.
